allennlp_models/modelcards/pair-classification-esim.json

```
@@ -1,6 +1,59 @@
 {
     "id": "pair-classification-esim",
     "registered_model_name": "esim",
+    "registered_predictor_name": "textual_entailment",
     "display_name": "Enhanced LSTM for Natural Language Inference",
-    "archive_file": "decomposable-attention-elmo-2020.04.09.tar.gz"
+    "archive_file": "decomposable-attention-elmo-2020.04.09.tar.gz", //FIX: this file is the same as one for decomposable...
```

I think this might be the correct archive file:

```suggestion
    "archive_file": "esim-elmo-2020.02.10.tar.gz"
```